src/Aquifer.API/Endpoints/Resources/Content/AssignEditor/Endpoint.cs

```diff
@@ -36,6 +36,19 @@ public override async Task HandleAsync(Request request, CancellationToken ct)
             ThrowError("Unable to assign to a user outside your company");
         }
 
+        if (request.AssignedReviewerUserId is not null) {
+            var reviewer = await dbContext.Users 
+                .AsTracking()
```

Comment:
I think we can remove the `AsTracking()` here since it's a quick check and we don't do anything with the response other than check if it's null or not.